demo/tiles/circles-page.html

```diff
@@ -54,35 +54,38 @@
     properties: {
 
       animationConfig: {
-        value: function() {
-          var circles = Polymer.dom(this.root).querySelectorAll('.circle');
-          var circlesArray = Array.prototype.slice.call(circles);
-          return {
-            'entry': [{
-              name: 'cascaded-animation',
-              animation: 'scale-up-animation',
-              nodes: circlesArray
-            }],
-
-            'exit': [{
-              name: 'hero-animation',
-              id: 'hero',
-              fromPage: this
-            }, {
-              name: 'cascaded-animation',
-              animation: 'scale-down-animation'
-            }]
-          };
-        }
+        type: Object
       }
     },
 
     listeners: {
       'click': '_onClick'
     },
 
+    attached: function() {
+      var circles = Polymer.dom(this.root).querySelectorAll('.circle');
```

Comment:
I'll implement as I believe it's clearer and promotes reusability, but this is only attached once